docs/commands.md

```diff
@@ -392,6 +394,21 @@ GRAPH.QUERY DEMO_GRAPH
 
 Extended `WITH` functionality is currently in development, see [known limitations](known_limitations.md).
 
+#### UNWIND
+The UNWIND cluase breaks down a given list into a squence of records, each contains a single element in the list.
+
+The order of the records preserve the original list order.
+
+```sh
+GRAPH.QUERY DEMO_GRAPH
+"CREATE (p {array:[1,2,3]})"
+```
+
+```sh
+GRAPH.QUERY DEMO_GRAPH
```

Comment:
Have you verified that this query works?

Reply:
yes